docs/docs-components/DevelopmentEditor.js

```diff
@@ -64,7 +64,11 @@ export default function DevelopmentEditor({ code }: {| code: ?string | (() => No
     // Remove export statement
     .replace('export default', '')
     // Add React. to React imports
-    .replace(reactRegex, 'React.$&');
+    .replace(reactRegex, 'React.$&')
+    .replace(
+      'const [showCmp, setShowCmp] = React.useState(true);',
```

Comment:
This isn't great — anyone adding new docs examples needs to know to follow this format or their page will break, and it isn't obvious — but I can't think of a better solution.

But I would recommend being more explicit here with `showComponent` and `setShowComponent`. "Cmp" isn't a widely-used abbreviation, and letters are free — no need to be economical.

Reply:
DONE